pkg/tanka/inline.go

```diff
@@ -45,12 +45,17 @@ func (i *InlineLoader) Load(path string, opts JsonnetOpts) (*v1alpha1.Environmen
 		return nil, fmt.Errorf("Found no environments in '%s'", path)
 	}
 
-	root, base, err := jpath.Dirs(path)
+	root, _, err := jpath.Dirs(path)
 	if err != nil {
 		return nil, err
 	}
 
-	name, err := filepath.Rel(root, base)
+	base, err := jpath.Entrypoint(path)
```

Comment:
```suggestion
	file, err := jpath.Entrypoint(path)
```

`jpath.Entrypoint` does not return a baseDirectory, but instead the absolute filename of the entrypoint